src/components/external-source/ExternalSourceManager.svelte

```diff
@@ -0,0 +1,1411 @@
+                }}
+              />
+            </fieldset>
+
+            <fieldset id="file-upload-fieldset">
+              <button
+                disabled={!parsed}
+                class="st-button w-100"
+                type="submit"
+                use:permissionHandler={{
+                  hasPermission: hasCreatePermission,
+                  permissionError: createPermissionError,
+                }}
+              >
+                {$creatingExternalSource ? 'Uploading...' : 'Upload'}
+              </button>
+              {#if parsed}
+                <div style="padding-top:10px">
+                  <button class="st-button w-100" type="reset">Dismiss</button>
+                </div>
+              {/if}
+            </fieldset>
+          </div>
+          <Field field={derivationGroupField}>
+            <label for="derivation-group" slot="label">Derivation Group</label>
+            <input
+              autocomplete="off"
+              class="st-input w-100"
+              name="derivation-group"
+              disabled={isDerivationGroupFieldDisabled}
+            />
+          </Field>
+          <Field field={keyField}>
+            <label for="key" slot="label">Key</label>
+            <input disabled bind:value={keyInputField} autocomplete="off" class="st-input w-100" name="key" required />
+          </Field>
+
+          <Field field={sourceTypeField}>
+            <label for="source-type" slot="label">Source Type</label>
+            <input disabled autocomplete="off" class="st-input w-100" name="source-type" required />
+          </Field>
+
+          <fieldset>
+            <DatePickerField
+              disabled={true}
+              field={startTimeDoyField}
+              label={`Start Time (${$plugins.time.primary.label}) - ${$plugins.time.primary.formatString}`}
+              name="start-time"
+            />
+          </fieldset>
+
+          <fieldset>
+            <DatePickerField
+              disabled={true}
+              field={endTimeDoyField}
+              label={`End Time (${$plugins.time.primary.label}) - ${$plugins.time.primary.formatString}`}
+              name="end_time"
+            />
+          </fieldset>
+
+          <fieldset>
+            <DatePickerField
+              disabled={true}
+              field={validAtDoyField}
+              label={`Valid At Time (${$plugins.time.primary.label}) - ${$plugins.time.primary.formatString}`}
+              name="valid_at"
+            />
+          </fieldset>
+        </form>
+      {/if}
+    </svelte:fragment>
+  </Panel>
+
+  <CssGridGutter track={1} type="column" />
+
+  <CssGrid rows={gridRowSizes}>
+    <!-- External Source Table -->
+    <Panel padBody={true}>
+      <svelte:fragment slot="header">
+        <slot name="left">
+          <SectionTitle><Truck />External Sources</SectionTitle>
+          <div class="filter">
+            <div class="timeline-editor-layer-filter">
+              <Input>
+                <input
```

Comment:
Any reason to use a complex menu here instead of a simple text filter on the table like the one used in the Plan page for filtering the activity table?
![image](https://github.com/user-attachments/assets/81344d25-44bd-4858-825a-7de7954e7807)

Reply:
The idea to use the `Menu` here was to make source type filtering mimic activity type filtering in the plan.

Reply:
Suggestion is to replace this filter with a filter that filters on all text in the table, like the one in `ActivityDirectivesTablePanel.svelte`.

Reply:
Addressed in [implement multi-delete](https://github.com/NASA-AMMOS/aerie-ui/pull/1396/commits/b1252a112fe88cee0f20629cfdf670e5c88bc204).